lti_consumer/tests/unit/test_api.py

```diff
@@ -1,16 +1,13 @@
 """
 Tests for LTI API.
 """
+from unittest.mock import Mock, patch
+
 from Cryptodome.PublicKey import RSA
 from django.test.testcases import TestCase
-from mock import Mock, patch
-
-from lti_consumer.api import (
-    _get_or_create_local_lti_config,
-    get_lti_consumer,
-    get_lti_1p3_launch_info,
-    get_lti_1p3_launch_start_url,
-)
+
+from lti_consumer.api import (_get_or_create_local_lti_config, get_lti_1p3_launch_info, get_lti_1p3_launch_start_url,
+                              get_lti_consumer)
```

Comment:
I liked the old indentation better.

Reply:
:+1: 

Me too, not sure why the change.

Reply:
Reverted the change that was made by the `pyugprade` package.